spec/controllers/settings/profiles_controller_spec.rb

```diff
@@ -23,5 +23,10 @@
       expect(account.reload.display_name).to eq 'New name'
       expect(response).to redirect_to(settings_profile_path)
     end
+
+    it 'renders :show if parameter is missing' do
+      put :update
+      expect(response).to render_template :show
```

Comment:
What's the thinking behind having this be the response?

I would think that either of these approaches...

- Return a 400 as an error because the params are not valid when it's missing, or;
- Consider the lack of params to be a request for an empty update, which will "succeed", so do nothing and then redirect like any other success

...would make more sense than forcing the show to render ... since you'd have to purposefully do something wrong to create this situation?